# Master's research project proposal: _On the genomic variability of inbred animal models and implications for medical research_
Mark Ziemann
2020-08-06

## Supervisory team

Dr Mark Ziemann will be primary supervisor and we will investigate candidates for a secondary supervisor.

## Background

Poor reproducibility of preclinical animal studies has been garnering greater attention from funding agencies and the general public who are concerned that a large portion of animal-based research funding is being wasted [1].
While a lack of rigour in data collection and reporting have been suggested, poor experimental design is also to blame [2].
Inter-individual phenotypic variation is a major consideration in experimental design and cost. Inbred mouse models are used because they are thought to exhibit less phenotypic variability, however in practise, variability levels in many endpoints are similar to outbred mice [3].
Experiments show that genetic variability can persist after over 40 generations of inbreeding and contribute to phenotypic variation [4,5].
Specific studies of LINE-1 elements in the common C57/Bl6J strain suggest that transposition events are as frequent as one in eight live births [6], providing another souce of genetic variation.

Previous genomic analysis of a medically relevant murine model by Dr Ziemann revealed substantial inter-individual variability in copy number at certain loci.
In particular, paired-end sequencing showed evidence of variable and widespread retro-transposition dynamics within the colony.
We suspect that genomic differences such as this contribute to gene expression and phenotypic variability, confounding in animal studies.
If the driving forces behind inter-individual variability can be identified, it will lead to more efficient research practises.
In this project, we will be mining public genomic datasets to find evidence for inter-individual genomic variability.

For further information the reader is referred to references [7-9].

## Hypothesis/research question

In this Masters project we want to address the question What evidence is there in publicly accessible databases to support the idea that inter-individual genomic variability in animal models exists and does this correlate with altered gene expression?

## Research approach

In order to address this question, we will be performing searches of datasets uploaded to NCBIâ€™s Short Read Archive [10].
Primarily we will focus on identifying genome sequence datasets derived from tissue samples of inbred mice, especially ones with matched gene expression profiles.
Sequence datasets will undergo genome alignment and analysis of different types of variants including SNVs, STRs SVs and transposon integration.
If available, we will be analysing matching gene expression datasets to highlight associations between genome variations and expression profiles.
If time permits, there is opportunity to build upon these findings by performing the same analyses in outbred mouse colonies. 

This project builds upon the previous work by Simranjit Kaur and Mark Ziemann who have performed a preliminary analysis of copy number variation in several datasets, but needs to be further developed.
Her thesis is available for your information.

## Project support and logistics

Dr Ziemann has in excess of 10 years genomics experience and will train the candidate to undertake computational analyses.
Dr Ziemann has access to computer systems for undertaking this work. There is no wet-lab component to this project, however the results from this work will be used to guide the design of follow-up experiments.
There will be weekly progress meetings with Dr Ziemann and monthly meetings with the entire supervisory team.

In the period before the start of the project, the candidate and supervisor will meet every two weeks to discuss the project.

The supervisory team will make all reasonable endeavours to support the candidate during the research including adjusting the contact time if necessary.
The supervisory team will promptly respond to correspondence on the next banking day.
The supervisory team will actively assist to solve technical issues such as computer code bugs to keep the project moving.

The candidate will contribute research via GitHub including code and thesis writing.
The project is compatible with the two trimester time-frame of Masters projects at Deakin.

## Expectations

For the duration of the project, the candidate is expected to work for 30 to 40 hours per week on the project.
That ime commitment includes research, reading, coding, writing, administrative tasks, relevant self-education and group activities.
The candidate will draw up a timeline for thesis completion including milestones for the completion of the research and writing of each chapter.
The candidate is expected to stick to their own timeline.
The candidate is entitled to two weeks time off from the project that can be taken at any time so long as the supervision team is notified at least one week beforehand.
The candidate will participate in the group activities including weekly meeting.

## References

1. Collins FS, Tabak LA. Policy: NIH plans to enhance reproducibility. Nature.
2014 Jan 30;505(7485):612-3. 

2. Pound P, Ebrahim S, Sandercock P, Bracken MB, Roberts I; Reviewing Animal Trials Systematically (RATS) Group. Where is the evidence that animal research benefits humans? BMJ. 2004 Feb 28;328(7438):514-7.

3. Jensen VS, Porsgaard T, Lykkesfeldt J, Hvid H. Rodent model choice has major impact on variability of standard preclinical readouts associated with diabetes and obesity research. Am J Transl Res. 2016 Aug 15;8(8):3574-84.

4. Watkins-Chow DE, Pavan WJ. Genomic copy number and expression variation within the C57BL/6J inbred mouse strain. Genome Res. 2008 Jan;18(1):60-6. Freedman LP, Cockburn IM, Simcoe TS. The Economics of Reproducibility in Preclinical Research. PLoS Biol. 2015 Jun 9;13(6):e1002165. 

5. Casellas J, Medrano JF. Within-generation mutation variance for litter size in
inbred mice. Genetics. 2008 Aug;179(4):2147-55. 

6. Richardson SR, Gerdes P, Gerhardt DJ, et al. Heritable L1 retrotransposition in the mouse primordial germline and early embryo. Genome Res. 2017;27(8):1395-1405. doi:10.1101/gr.219022.116

7. Freedman LP, Cockburn IM, Simcoe TS. The Economics of Reproducibility in Preclinical Research. PLoS Biol. 2015 Jun 9;13(6):e1002165. 

8. Bracken MB. Why animal studies are often poor predictors of human reactions to exposure. J R Soc Med. 2009 Mar;102(3):120-2. 

9. Rivera J, Tessarollo L. Genetic background and the dilemma of translating mouse studies to humans. Immunity. 2008 Jan;28(1):1-4. 

10. Kodama Y, Shumway M, Leinonen R; International Nucleotide Sequence Database Collaboration. The Sequence Read Archive: explosive growth of sequencing data. Nucleic Acids Res. 2012 Jan;40(Database issue):D54-6. 
